docs/fiddles/features/window-customization/native-window-controls/main.js

```diff
@@ -0,0 +1,15 @@
+const { app, BrowserWindow } = require('electron')
+
+function createWindow () {
+  const win = new BrowserWindow({
+    // remove the default titlebar
+    titleBarStyle: 'hidden',
+    // expose window controlls in Windows/Linux
+    ...(process.platform !== 'darwin' ? { titleBarOverlay: true } : {})
```

Comment:
Is it bad to have `titleBarOverlay: true` active on macOS, or are we explicitly removing it because it's a no-op?

Reply:
Removing it bc it's a no-op. It could also be a nice reminder that you can have different browserWindow configurations for per platform